services/mealPlans.service.js

```diff
@@ -0,0 +1,9 @@
+const MealPlan = require("../models/meal-plan.model");
+
+const getMealPlans = async (user_id) => {
+    return await MealPlan.find({});
```

Comment:
I would recomend to assign await result to the variable and then return that variable